plugins/PlotSquared/PlotSquared.svg

```diff
@@ -1,50 +1,50 @@
+		<metadata id="metadata8">
+			<rdf:RDF>
+				<cc:Work rdf:about="">
+					<dc:format>image/svg+xml</dc:format>
+                    <dc:type rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+				</cc:Work>
+			</rdf:RDF>
+		</metadata>
+    <defs id="defs6">
+			<clipPath clipPathUnits="userSpaceOnUse" id="clipPath18">
+				<path d="M 0,2500 H 3000 V 0 H 0 Z" id="path16" />
+			</clipPath>
+		</defs>
+    <sodipodi:namedview pagecolor="#ffffff" bordercolor="#666666" borderopacity="1" objecttolerance="10" gridtolerance="10" guidetolerance="10" inkscape:pageopacity="0" inkscape:pageshadow="2" inkscape:window-width="640" inkscape:window-height="480" id="namedview4" />
+    <g id="g10" inkscape:groupmode="layer" inkscape:label="PlotSquared" transform="matrix(1.3333333,0,0,-1.3333333,0,3333.3333)">
+			<g id="g12">
+				<g id="g14" clip-path="url(#clipPath18)">
+					<g id="g20" transform="translate(1486.1511,2242.6453)">
+						<path d="m 0,0 c 16.533,10.846 33.211,21.453 50.104,31.699 78.972,-48.281 153.985,-102.704 227.269,-159.144 148.61,-115.422 287.884,-243.01 414.393,-382.333 111.39,-122.861 212.751,-255.152 298.898,-396.971 52.744,-87.322 100.544,-177.884 139.514,-272.214 -11.638,-3.551 -23.108,-7.655 -34.362,-12.286 l -0.24,0.288 c -11.135,12.982 -24.141,24.212 -34.915,37.506 -22.557,23.013 -45.425,45.737 -68.03,68.678 -19.725,20.253 -40.601,39.45 -58.958,60.974 -36.355,36.451 -72.517,73.093 -108.944,109.471 -22.628,26.013 -48.064,49.385 -71.965,74.197 -19.029,19.485 -38.706,38.346 -57.519,57.999 -12.166,14.998 -26.684,27.716 -39.93,41.658 -27.668,27.524 -54.903,55.479 -82.571,82.979 -23.924,27.956 -51.664,52.264 -76.692,79.164 -4.68,4.487 -8.855,10.774 -15.886,11.326 -22.34,34.027 -58.311,57.327 -97.377,67.502 -104.312,99.153 -215.487,191.202 -332.661,274.782 -117.942,-83.94 -229.476,-176.781 -334.484,-276.39 -26.684,-0.024 -53.368,0.024 -80.076,-0.024 0.024,-26.564 0.048,-53.104 0,-79.668 -72.229,-73.021 -139.491,-150.937 -202.385,-232.092 -63.758,-82.619 -121.973,-169.51 -173.541,-260.264 131.932,-69.061 257.864,-149.521 375.926,-240.275 0.096,-26.444 -0.12,-52.888 0.096,-79.332 l 0.744,-0.984 c 20.109,-24.14 43.409,-45.233 65.126,-67.861 15.118,-15.382 30.571,-30.404 45.569,-45.881 17.565,-20.733 37.698,-39.042 56.607,-58.503 19.917,-20.781 41.25,-40.218 59.967,-62.151 29.156,-29.299 58.167,-58.815 87.515,-87.922 29.155,-33.043 61.502,-63.111 92.169,-94.738 13.726,-12.67 25.124,-27.571 38.634,-40.457 25.029,-25.365 50.129,-50.657 75.325,-75.853 -37.914,-51.208 -73.741,-103.952 -107.192,-158.183 -167.83,273.317 -397.235,507.305 -662.37,687.158 -81.875,55.335 -167.23,105.584 -255.681,149.641 -52.815,26.276 -106.831,50.248 -162.239,70.381 99.393,233.628 242.795,446.715 410.289,636.79 93.562,106.088 194.634,205.433 301.466,298.13 C -217.335,-155.808 -111.439,-73.789 0,0" style="fill:#062f4c;fill-opacity:1;fill-rule:nonzero;stroke:none" id="path22" />

```

Comment:
```svg
<path d="M 0 0 C 16.533 10.846 33.211 21.453 50.104 31.699 C 129.076 -16.582 204.089 -71.005 277.373 -127.445 C 425.983 -242.867 565.257 -370.455 691.766 -509.778 C 803.156 -632.639 904.517 -764.93 990.664 -906.749 C 1043.408 -994.071 1091.208 -1084.633 1130.178 -1178.963 C 1118.54 -1182.514 1106.692 -1186.384 1095.438 -1191.015 C 1095.438 -1191.015 768.153 -871.756 744.252 -846.944 C 725.223 -827.459 421.855 -509.682 382.789 -499.507 C 278.477 -400.354 167.302 -308.305 50.128 -224.725 C -67.814 -308.665 -179.348 -401.506 -284.356 -501.115 C -311.04 -501.139 -337.724 -501.091 -364.432 -501.139 C -364.408 -527.703 -364.384 -554.243 -364.432 -580.807 C -436.661 -653.828 -503.923 -731.744 -566.817 -812.899 C -630.575 -895.518 -688.79 -982.409 -740.358 -1073.163 C -608.426 -1142.224 -482.494 -1222.684 -364.432 -1313.438 C -364.336 -1339.882 132.124 -1901.924 157.32 -1927.12 C 119.406 -1978.328 83.579 -2031.072 50.128 -2085.303 C -117.702 -1811.986 -347.107 -1577.998 -612.242 -1398.145 C -694.117 -1342.81 -779.472 -1292.561 -867.923 -1248.504 C -920.738 -1222.228 -974.754 -1198.256 -1030.162 -1178.123 C -930.769 -944.495 -787.367 -731.408 -619.873 -541.333 C -526.311 -435.245 -425.239 -335.9 -318.407 -243.203 C -217.335 -155.808 -111.439 -73.789 0 0" style="fill:#062f4c;fill-opacity:1;fill-rule:nonzero;stroke:none" id="path22"></path>
```

Fixes the white line mentioned in discord by removing a few (unnecessary) path nodes and therefor providing a consistent background

Reply:
Thanks a lot!